arm-sql/2015-05-01-preview/swagger/automaticTuning.json

```diff
@@ -258,7 +227,10 @@
             "Inherit",
             "Custom"
           ],
-          "type": "string"
+          "type": "string",
+          "x-ms-enum": {
+            "name": "AutomaticTuningMode"
```

Comment:
Got it.  We have a required property in our generator attribute where it adds modelAsString:true if true, and nothing if false.  I did it that way because it seemed to be the precedent, but I see how it might seem like we aren't fully aware of the property.

I checked in a change to our Swashbuckle project to do explicit modelAsString:false to make this more clear in our docs.  Thanks, Veronica.

@tomapaunovic you probably already did this, but if you haven't, please make sure all the enums not marked modelAsString are not likely to change.